app/src/test/kotlin/com/intive/picover/common/viewmodel/state/ViewModelStateTest.kt

```diff
@@ -11,7 +11,7 @@ internal class ViewModelStateTest : ShouldSpec(
 	{
 		val data: Any = mockk()
 
-		should("SHOULD isLoading AND isError AND isLoaded AND return values based on view model state") {
```

Comment:
agreed on the meeting that we should follow avoiding "SHOULD" in the tests names